### DISCUSSION TITLE
Not insert new data, why the storage size increase so largely from 14G to 100+GB during searching usage?

### DISCUSSION THREAD

**Discussion by rocke2020** [milvus-io/milvus#17388] [Q&A and General discussion] [answered]
I use milvus 2.0.2, standalone, I created a collection and insert data and create index, that's initialization.
After initialization, the local volumes is 14GB. I did not insert any new data later.
Then I only do frequent search for 1 week, the volumes stoarge increase to more than 100GB, and it is contintouls increase the storage size when I am searching.
When not insert new data, why the storage size increase so largely?
I find local volumes/milvus/rdb_data which is linked to milvus docker service.
There are 1688 new increased *.sst files. These *.sst files is auto increased during my searching process.
Could you explain what's the purpose of the *.sst files? Could we auto delete these *.sst files by configuration?
Could we limit the local storage size when not insering new data, but only do searching?

By the way, my log config. 
log:
  level: debug # Only supports debug, info, warn, error, panic, or fatal. Default 'info'.
  file:
    rootPath: "" # default to stdout, stderr
    maxSize: 300 # MB
    maxAge: 10 # Maximum time for log retention in day.
    maxBackups: 20
  format: text # text/json

thanks!!

**Comment by yhmo** [comment, marked as answer]
Same question with this issue: https://github.com/milvus-io/milvus/issues/16606
